pkg/test/framework/README.md

```diff
@@ -54,10 +54,6 @@ func TestHTTP(t *testing.T) {
     
     be := b.EndpointsForProtocol(model.ProtocolHTTP)[0]
     result := a.CallOrFail(be, components.AppCallOptions{}, t)[0]
-
```

Comment:
probably don't worry about changing this file.  I believe it's going away anyway.

@ozevren?